src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -0,0 +1,206 @@
+
+            <StyledLabelBoldLarge>
+              {t('phrasalTemplateItem.phraseOrder', { order: index + 1 })}
+            </StyledLabelBoldLarge>
+
+            <Tooltip tooltipTitle={t('phrasalTemplateItem.titleHint')}>
+              <IconButton
+                onClick={(e) => {
+                  e.stopPropagation();
+                }}
+                type="button"
+              >
+                <Svg
+                  aria-label={t('phrasalTemplateItem.titleHintLabel')}
+                  fill={variables.palette.outline}
+                  height={18}
+                  id="help"
+                  width={18}
+                />
+              </IconButton>
+            </Tooltip>
+          </StyledFlexTopCenter>
+
+          <StyledFlexTopCenter sx={{ gap: 0.8 }}>
+            <Button
+              onClick={handlePreviewPhrase}
+              sx={{ gap: 0.8, width: 'max-content' }}
+              variant="text"
+              disabled={formState.isDirty || formState.isSubmitting || !formState.isValid}
+            >
+              <Svg aria-hidden height={18} id="notes" width={18} />
+              {t('phrasalTemplateItem.btnPreviewPhrase')}
+            </Button>
+
+            <IconButton color="default" disabled={!canRemovePhrase} onClick={handleRemovePhrase}>
+              <Svg
+                aria-label={t('phrasalTemplateItem.btnRemovePhrase')}
+                fill="currentColor"
+                id="trash"
+              />
+            </IconButton>
+          </StyledFlexTopCenter>
+        </StyledFlexSpaceBetween>
+      </StyledAccordionSummary>
+
+      <StyledAccordionDetails>
+        <PhrasalTemplateImageField
+          value={imageFieldValue}
+          onChangeValue={(value = '') => setValue(`${name}.image`, value || '')}
+        />
+
+        <StyledPhraseTemplateFieldSet>
+          {fields.map((field, i) => {
+            const isOnlySentenceField =
+              field.type === 'sentence' &&
+              fields.filter(({ type }) => type === 'sentence').length < 2;
+            const isOnlyResponseField =
+              field.type === 'item_response' &&
+              fields.filter(({ type }) => type === 'item_response').length < 2;
+            const hasMinimumFields = fields.length > 2;
+
+            return (
+              <PhrasalTemplateField
+                canRemove={hasMinimumFields && !isOnlyResponseField && !isOnlySentenceField}
+                key={field.id}
+                name={`${phraseFieldsName}.${i}`}
+                onRemove={() => {
+                  remove(i);
+                }}
+                responseOptions={responseOptions}
+                type={field.type}
+                placeholder={fieldPlaceholders[i]}
+              />
+            );
+          })}
+        </StyledPhraseTemplateFieldSet>
+
+        <Button
+          aria-expanded={open}
+          aria-haspopup="true"
+          onClick={handlePressAdd}
+          sx={{ gap: 0.8, width: 'max-content' }}
```

Comment:
This button should actually be shifted left a bit, to have a total gap of 16 pixels from the left container edge:
```suggestion
          sx={{ gap: 0.8, ml: -1.4, width: 'max-content' }}
```